MSAL/src/MSALPublicClientApplication.m

```diff
@@ -644,18 +660,37 @@ - (void)acquireTokenForScopes:(NSArray<NSString *> *)scopes
 - (void)acquireTokenSilentWithParameters:(MSALSilentTokenParameters *)parameters
                          completionBlock:(MSALCompletionBlock)completionBlock
 {
+    __auto_type block = ^(MSALResult *result, NSError *msidError, id<MSIDRequestContext> context)
+    {
+        NSError *msalError = [MSALErrorConverter msalErrorFromMsidError:msidError classifyErrors:YES msalOauth2Provider:self.msalOauth2Provider];
```

Comment:
should we check that msidError is present before calling conversion?

Reply:
msalErrorFromMsidError:msidError returns nil when we provide nil error to it, so this is not required.